buildservice.proto

```diff
@@ -30,6 +30,7 @@ service BuildService {
   rpc Package(stream FileNode) returns (BuildResponse) {} 
   rpc CreatePackage(PackageNode) returns (BuildResponse) {}
   rpc CreateProject(ProjectNode) returns (BuildResponse) {}
+  rpc Project(stream PackageNode) returns (BuildResponse) {}
```

Comment:
Chose a name that uses a verb to reflect what this function does.

Reply:
@hemarkus 
Ok. But we have to follow a generic syntax. I was trying to follow the structure that we use until now. Either I could name it Project (see Package function above) or SendProjectNode. But I chose UpdateProjectNode. I believe is more appropriate and we should follow the 'Update...Node' structure in the future.